src/ObjectParser.js

```diff
@@ -42,7 +42,15 @@ export default class ObjectParser {
       if (Array.isArray(value)) {
         const val = value[0];
         if (Array.isArray(val)) return ['JSON'];
-        return [(this.getFieldConfig(val): any)];
+
+        const args =
+          opts && opts.typeName && opts.fieldName
+            ? {
+                typeName: `${opts.typeName}_${upperFirst(opts.fieldName)}`,
+                fieldName: '0',
```

Comment:
Please rewrite this method on
```
{
typeName: opts.typeName
fieldName: opts.fieldName
}
```

`Person_Limbs` is better name for typename rather than `Person_Limbs_0`